docs/data/material/migration/migration-v4/v5-component-changes.md

```diff
@@ -1615,6 +1615,7 @@ The `input` key is applied on another element.
 Change the default `indicatorColor` and `textColor` prop values to "primary".
 
 This is done to match the most common use cases with Material Design.
+Here is how to keep previouse colors.
```

Comment:
This would be a little more explicit, and I think helps to explain the changes:

```suggestion

If you'd prefer to keep the v4 color styles, use `"secondary"` and `"inherit"`, respectively, as shown below:
```